genshin/client/components/base.py

```diff
@@ -356,6 +356,58 @@ async def request_webstatic(
 
         return data
 
+    def _get_ds_headers(
+        self,
+        region: types.Region,
+        data: typing.Any = None,
+        params: typing.Optional[typing.Mapping[str, typing.Any]] = None,
+        lang: typing.Optional[str] = None,
+    ) -> typing.Dict[str, typing.Any]:
```

Comment:
This should go into `utility/ds.py` since it doesn't depend on the client.

Reply:
Fixed.